### DISCUSSION TITLE
react ts在Menubar提示不存在children

### DISCUSSION THREAD

**Discussion by BrickerZou** [antvis/X6#2092] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/73508461/167281400-45e33789-eb48-4f33-9bfa-227a46636d9f.png)
react似乎把Menubar标签内的当成children属性传递了，但是Menubar接口里并不存在这一属性，这该怎么解决

代码是直接复制的antv x6 react样例，但用的react最新版本
https://codesandbox.io/s/fy7ysk?file=/src/app.tsx:504-557

**Comment by NewByVector** [comment, marked as answer]
React 18 需要在 props 里面指定 children 属性，我们 2.0 版本会兼容 React 18
